### DISCUSSION TITLE
does the omc.exe can translate a tpl-file to c-file directly?

### DISCUSSION THREAD

**Discussion by pily1** [OpenModelica/OpenModelica#12430] [Q&A] [answered]
I konw the omc.exe can translate a tpl-file to mo-file,the mo-file can translate to a c-file,
does the omc.exe can translate a tpl-file to c-file directly?
if can,how to do this?what's the command?

**Comment by perost** [comment, marked as answer]
No, it's done in two steps, tpl -> mo -> c. I'm not sure exactly how it's done since it's never done manually but only as part of building the compiler, so you'd have to check the CMake or Make files if you want to know the exact commands that are used.
